index.html

```diff
@@ -0,0 +1,103 @@
+            <div class="fields" >
+              <h3 for="">Would you recommend freeCodeCamp to a friend?</h3>
+              <input type="radio" name="Definitely" required>
+              <label for="">Definitely</label>
+              <input type="radio" name="Maybe">
+              <label for="">Maybe</label>
+              <input type="radio" name="Not Sure">
+              <label for="">Not Sure</label>
+            </div>
+            <div class="fields">
+              <h3 for="">What is your favorite feature of freeCodeCamp?</>
+              <select name="dropdown" id="dropdown" required>
+                <option value="" disabled>Select an Option</option>
+                <option value="Student">Challenges</option>
+                <option value="Fulltime Job">Projects</option>
+                <option value="Fulltime Learner">Community</option>
+                <option value="Prefer Not to say">Open Source</option>
+              </select>
+            </div>
+            <div class="fields">
+              <h3>What would you like to see improved? (Check all that apply)</h3>
+              <input type="checkbox" id="Front-end Projects" name="Front-end Projects" value="Front-end Projects">
+              <label for="vehicle1"> Front-end Projects</label>
+              <input type="checkbox" id="Back-end Projects" name="Back-end Projects" value="Back-end Projects">
+              <label for="vehicle2"> Back-end Projects</label>
+              <input type="checkbox" id="Data Visualization" name="Data Visualization" value="Data Visualization">
+              <label for="Data Visualization"> Data Visualization</label><br><br>
+              <input type="checkbox" id="Challenges" name="Challenges" value="Challenges">
+              <label for="Challenges"> Challenges</label><br><br>
+              <input type="checkbox" id="Open Source Community" name="Open Source Community" value="Open Source Community">
+              <label for="Open Source Community"> Open Source Community</label><br><br>
+              <input type="checkbox" id="Gitter help rooms" name="Gitter help rooms" value="Gitter help rooms">
+              <label for="Gitter help rooms"> Gitter help rooms</label><br><br>
+              <input type="checkbox" id="Videos" name="Videos" value="Videos">
+              <label for="Videos"> Videos</label><br><br>
+              <input type="checkbox" id="City Meetups" name="City Meetups" value="City Meetups">
+              <label for="City Meetups"> City Meetups</label><br><br>
+              <input type="checkbox" id="Forum" name="Forum" value="Forum">
+              <label for="Forum"> Forum</label><br><br>
+              <input type="checkbox" id="Wiki" name="Wiki" value="Wiki">
+              <label for="Wiki"> Wiki</label><br><br>
+              <input type="checkbox" id="Additional Courses" name="Additional Courses" value="Additional Courses">
+              <label for="Wiki"> Additional Courses</label><br><br>
```

Comment:
@name Great job implementing the fields section 👏🏽 However, kindly apply flex properties to the checkbox element so that the labels and the checkbox are aligned in the same line and also give the elements the same amount of margin to maintain consistency.